.travis.yml

```diff
@@ -58,6 +58,8 @@ install:
   - ${DOC} git clone --quiet https://github.com/Pyomo/pyomo-model-libraries.git
  # Install PyUtilib (master branch)
   - ${DOC} pip install --quiet git+https://github.com/PyUtilib/pyutilib
+ # Install z3 Satisfiability solver
+  - ${DOC} pip install --quiet z3-solver
```

Comment:
We should not install z3 here.  Instead, this should be added to the Docker -- and REMOVED for SLIM builds.  I will go ahead and put in the PR to get z3 added to the Docker image.  What is the plan for Appveyor?  Does it not (cleanly) support z3?

Reply:
Thanks John. I was having a bit of difficulty getting Z3 to properly install on appveyor, either with pip or conda, as suggested

Reply:
For the record, I am OK only testing this on Travis/Jenkins -  was just wondering what the final verdict for Appveyor was.